lang/de/miquiz.php

```diff
@@ -5,31 +5,36 @@
 $string['modulename'] = 'MI-Quiz';
 $string['modulenameplural'] = 'MI-Quizze';
 
-//$instance_name ='';
 $instance_name = get_config('mod_miquiz', 'instancename');
 $string['modulename_help'] = 'Erstelle ein ' . $instance_name . ' Duell.';
 $additional_info = get_config('mod_miquiz', 'additional_info');
 if (!empty($additional_info)) {
   $string['modulename_help'] .= $additional_info;
 }
 
+//strings for Administration->Manage roles page 
+$string['miquiz:overview'] = 'Globale ' . $instance_name . ' Aktivitäts&uuml;bersicht';
+$string['miquiz:addinstance'] = 'Erstelle ein ' . $instance_name . ' Duell';
+
+$string['miquiz_index_title_overview'] = $instance_name .' &Uuml;bersicht';
 $string['miquiz_index_title'] = $instance_name .' Kurs &Uuml;bersicht';
 $string['miquiz_index_reports'] = 'Anzahl Reports';
 $string['miquiz_index_download'] = 'Statistiken herunterladen';
 $string['miquiz_index_noquizselected'] = 'Bitte w&auml;hlen sie mindestens ein Quiz aus!';
+$string['miquiz_index_overview'] = 'Zur globalen &Uuml;bersicht';
 
-$string['miquiz_setting_info_title'] = 'Zusätzliche Info';
-$string['miquiz_setting_info_helper'] = 'Erscheint beim Erstellen der Aktivität';
+$string['miquiz_setting_info_title'] = 'Zus&auml;tzliche Info';
```

Comment:
Did you experienced a problem without html entities? On the ecampus and my local moodle everything works fine without html entities.

Reply:
I experience issues with the old Moodle version we used. Just changed it to be consistent.